src/components/RecentThoughts.jsx

```diff
@@ -0,0 +1,27 @@
+import { Like } from "./Like";
+import moment from "moment";
+
+// RecentThoughts Component
+export const RecentThoughts = ({ thoughts, likes, handleLike }) => {
+  // Function to get how many seconds, minutes or hours ago it was posted
+
+  const timeAgo = (createdAt) => {
+    return moment(createdAt).fromNow();
+  };
+
+  //Function to render thoughts
+  const renderRecentThoughts = () => {
+    return thoughts.map((thought) => (
+      <li key={thought._id} className="thought">
+        <p>{thought.message}</p>
```

Comment:
long messages without breaks (like "HELLOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO"), just continues outside of the happy thought card. To prevent this, add 

overflow-wrap: break-word

to the css for this <p> :)